test/k8sT/Updates.go

```diff
@@ -70,31 +84,38 @@ var _ = Describe("K8sUpdates", func() {
 	})
 
 	BeforeEach(func() {
-		kubectl.Exec("sudo docker rmi cilium/cilium")
```

Comment:
There is a reason for this, if not the `cilium/cilium` image is  when we do `make docker-image`, now with docker.io prefix+tags will be ok, but maybe I would keep it.

Reply:
Discussed offline